nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/spectra/StripController.java

```diff
@@ -181,64 +162,84 @@ public void initialize(VBox vBox) {
         Label startLabel = new Label("Start:");
         toolBar.getItems().add(startLabel);
         posSlider.setBlockIncrement(1);
+        posSlider.setMajorTickUnit(10);
         posSlider.setSnapToTicks(true);
         posSlider.setMinWidth(250);
-        toolBar.getItems().add(posSlider);
+        posSlider.setShowTickLabels(true);
+        posSlider.setShowTickLabels(true);
```

Comment:
duplicated line of code
```suggestion
        posSlider.setShowTickLabels(true);
```